atomium/template.php

```diff
@@ -23,6 +23,9 @@
 include_once __DIR__ . '/includes/common.inc';
 include_once __DIR__ . '/includes/config.inc';
 include_once __DIR__ . '/includes/preprocess.inc';
+include_once __DIR__ . '/../src/AttributesContainer.php';
+include_once __DIR__ . '/../src/AtomiumClassAttribute.php';
+include_once __DIR__ . '/../src/AtomiumPlaceholderAttribute.php';
```

Comment:
As composer_manager doesn't detect the composer files in teams then the solution has to be by injecting the files manually.